_internals/configuration.md

```diff
@@ -118,329 +118,329 @@ When `ANSI_DEFAULTS` is enabled, it enables the following ISO settings:
 
 
-### babelfishpg_tsql.escape_hatch_login_password_unlock
+### `babelfishpg_tsql.escape_hatch_login_password_unlock`
 
 Escape hatch for login passwords unlock option.
 
 
-### babelfishpg_tsql.escape_hatch_nocheck_add_constraint
+### `babelfishpg_tsql.escape_hatch_nocheck_add_constraint`
 
 Escape hatch for `WITH [NO]CHECK` option in `ALTER TABLE ADD`
 
 
-### babelfishpg_tsql.escape_hatch_nocheck_existing_constraint
+### `babelfishpg_tsql.escape_hatch_nocheck_existing_constraint`
 
 Escape hatch for `WITH [NO]CHECK` option in `ALTER TABLE` on existing
 constraint.
 
 
-### babelfishpg_tsql.escape_hatch_query_hints
+### `babelfishpg_tsql.escape_hatch_query_hints`
 
 Escape hatch for query hints.
 
 
-### babelfishpg_tsql.escape_hatch_rowguidcol_column
+### `babelfishpg_tsql.escape_hatch_rowguidcol_column`
 
 Escape hatch for `ROWGUIDCOL` option
 
 
-### babelfishpg_tsql.escape_hatch_schemabinding_function
+### `babelfishpg_tsql.escape_hatch_schemabinding_function`
 
 Escape hatch for `SCHEMABINDING` option in `CREATE FUNCTION`.
 
 
-### babelfishpg_tsql.escape_hatch_schemabinding_procedure
+### `babelfishpg_tsql.escape_hatch_schemabinding_procedure`
 
 Escape hatch for `SCHEMABINDING` option in `CREATE PROCEDURE`.
 
 
-### babelfishpg_tsql.escape_hatch_schemabinding_trigger
+### `babelfishpg_tsql.escape_hatch_schemabinding_trigger`
 
 Escape hatch for `SCHEMABINDING` option in `CREATE TRIGGER`.
 
 
-### babelfishpg_tsql.escape_hatch_schemabinding_view
+### `babelfishpg_tsql.escape_hatch_schemabinding_view`
 
 Escape hatch for `SCHEMABINDING` option in `CREATE VIEW`.
 
 
-### babelfishpg_tsql.escape_hatch_session_settings
+### `babelfishpg_tsql.escape_hatch_session_settings`
 
 Escape hatch for session settings.
 
 
-### babelfishpg_tsql.escape_hatch_storage_on_partition
+### `babelfishpg_tsql.escape_hatch_storage_on_partition`
 
-Escape hatch for `storage_on_partition` option in 
+Escape hatch for `storage_on_partition` option in
 `CREATE/ALTER TABLE` and `CREATE INDEX`.
 
 
-### babelfishpg_tsql.escape_hatch_storage_options
+### `babelfishpg_tsql.escape_hatch_storage_options`
 
 Escape hatch for storage options option in `CREATE/ALTER TABLE/INDEX`.
 
 
-### babelfishpg_tsql.escape_hatch_table_hints
+### `babelfishpg_tsql.escape_hatch_table_hints`
 
 Escape hatch for table hints.
 
 
-### babelfishpg_tsql.escape_hatch_unique_constraint
+### `babelfishpg_tsql.escape_hatch_unique_constraint`
 
 Escape hatch for unique constraints.
 
 
-### babelfishpg_tsql.fmtonly
+### `babelfishpg_tsql.fmtonly`
 
 SQL-Server compatibility `FMTONLY` option.
 
 
-### babelfishpg_tsql.implicit_transactions
+### `babelfishpg_tsql.implicit_transactions`
 
 Enable implicit transactions.
 If no transaction is running, start an implicit transaction
 for qualified commands when `implicit_transactions` config
 option is on.
 
 
-### babelfishpg_tsql.language
+### `babelfishpg_tsql.language`
 
-T-SQL compatibility `LANGUAGE` option.
+T-SQL compatibility language option.
```

Comment:
There is no such option. This matches various t-SQL statements which have a language clause.

Reply:
Please explain that. I don't understand your comment.